### PR TITLE
Updating allowlisting

### PR DESCRIPTION
**Pull request by ierceg**

### CODE REVIEW

**Comment by RaeesBhatti** [comment]
It would be great if we mentioned as a canveat that IPs may be different for different accounts. @ierceg

**Comment by ierceg** [comment]
> It would be great if we mentioned as a canveat that IPs may be different for different accounts. @ierceg

It's already there:

> Note that your IP addresses may differ from the ones shown above.

But I will stress it out more.

**Comment by ierceg** [comment]
@RaeesBhatti please check now.

File: src/pages/guides/production.md [hunk 1]
```diff
@@ -119,7 +119,7 @@ mywebsite.xyz.        599    IN    A        208.69.180.14
 
 ### Allowing {{ PRODUCT_NAME }} IP Addresses {/*allowing-layer0-ip-addresses*/}
 
-Before going live, ensure that all {{ PRODUCT_NAME }} IP addresses are allowed in the security layer in front of your origin and/or API servers. The IP addresses you need to allow can be found on the _IP Whitelist_ section of the _Networking_ tab. Note that your IP addresses may differ from the ones shown above.
+Before going live, ensure that all {{ PRODUCT_NAME }} IP addresses are allowed in the security layer in front of your origin and/or API servers. The IP addresses you need to allow can be found on the _Allowlisting_ section of the _Networking_ tab of the _Environment_ page. Note that each team may have their own set of IPs so these values cannot be copied from one team to another.
```
Comment (RaeesBhatti) [inline src/pages/guides/production.md:122]: 🙌🏽

**Comment by RaeesBhatti** [comment]
Looks good to me!

**Pull request merged**